src/SourceBuild/Arcade/tools/SourceBuildArcadeTarball.targets

```diff
@@ -329,6 +334,8 @@
       Overwrite="true" />
 
     <MakeDir Directories="$(TarballRootDir)packages/prebuilt" />
```

Comment:
I've never been fond of this folder existing in the tarball.  It gives a bad impression and is an unnecessary attention getter.  A better approach would be to conditionally add the NuGet source based on the existence of the folder.  

Given the time constraints, I accept this change, but I think an issue should be logged to make this folder optional and remove it from the default tarball.